packages/chatbot-server-mongodb-public/src/tools/fetchPage.ts

```diff
@@ -34,7 +35,9 @@ export type FetchPageTool = Tool<
 
 export function makeFetchPageTool(
```

Comment:
pls make the constructor an object. more easy extensibility + easier to read

eg

```ts
export interface MakeFetchPageToolParams {
  // ...
}
```

Reply:
this pattern generally preferable for exported functions, such as this one. easier to use when invoking